### DISCUSSION TITLE
Display flickering with Pump S100101

### DISCUSSION THREAD

**Discussion by brucke** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#591] [Q&A]
Hey Community 😄  

yesterday was the day and i tested my board with my pump S100101 but it doen't work. 
If i connect the Board the display is flickering. 

My Pool: WHIRLPOOL MIAMI 60001 with Pump S100101

At the moment i have no idea what the problem is. Do i need a differen LLC (the red one), or some resistors between LLC and Display? 
It would be cool, if somebody has an tip for me what i should check! 😄  Thanks

## My Hardware Settings:
![PXL_20230812_090459861](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/6576396/65edfd9a-ec6f-4001-a645-325f38c51741)

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/6576396/9544be89-d074-461f-b214-d9255c096d1d)

## The Pump 
![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/6576396/8fdb3099-2d7e-4e78-8034-e6b8f12d8d59)

![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/6576396/4a9b1dfc-ca2a-481c-8665-5f67655f8511)

## The Problem:
https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/6576396/5f79a4fb-e0a3-4437-ab37-b0ba61201fa9

**Comment by brucke** [comment]
I found following Post 

https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/discussions/559

He used the Low Pass Filter. I ordered the required parts, lets see If thus solves the Problem :) 
But If Somebody hast additional Information let me know

**Comment by brucke** [comment]
The LowPass Filter is the solution now it works :)
Thanks a lot for this greate project :)

**Reply by visualapproach** [reply]
Great 👍🏼

**Reply by Alexxx2005** [reply]
Hey visualapproach,

nice Project, the Low Pass Filter is ONLY on C2,3,5 & 6 and R2,3,5 & 6 
is this correct.

best regards Alex

**Reply by visualapproach** [reply]
@Alexxx2005 Thanks, depends on your model, but all data lines. Not 5v, gnd or audio. Guess you have a four wire and then yes.

**Reply by Alexxx2005** [reply]
No it is the S100101 , i read Page 10 in the PDF with only C2,3,5 & 6 and R2,3,5 & 6

**Comment by Alexxx2005** [comment]
Hey Brucke,

nice Case would you like to share it ?

best regards Alex

**Reply by brucke** [reply]
Hey @Alexxx2005 , 

sorry for the delay. But i didn't receive a notification :)

I found the case on thingiverse but it is no longer available. 
But i found 3 cases maybe one fit your needs
- https://www.thingiverse.com/thing:5380738
- https://www.thingiverse.com/thing:5379037
- https://www.thingiverse.com/thing:6030872

Brucke

**Comment by visualapproach** [comment]
> No it is the S100101 , i read Page 10 in the PDF with only C2,3,5 & 6 and R2,3,5 & 6

That is under "build 4-wire" header... So C2 trough C7 (C8 is just filtering the audio) and R2-7

**Reply by Alexxx2005** [reply]
Ok Thank you , i understand it.
Do you have the STL from the from the first Post ?

**Reply by Alexxx2005** [reply]
Hi Visualapproach, unfortunately my device doesn't work on the S100101 Pump. I read a line must be crossed ? Is that so? I tried it with the red and the blue LLC. But nothing works and no beep when i plugging in the power plug. do you have any idea ? I must change the PIN Out assignment ? Thank You Greetings Alex

**Reply by visualapproach** [reply]
@Alexxx2005 What PCB do you have?
V2 -> you can rearrange some of the wires and set as V2B. Se manual.
V2B -> no need to rearrange wires. Set as V2B
In both cases you must use the rightmost ports on the PCB to have the RC filter work.

BTW "doesn't work" is not a helpful statement if you want me to guess your problem.
